### PR TITLE
#6685: added public apigateway timeout response

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]
Nice! Should this also apply to the async lambda or websockets?

**Comment by kkoskelin** [comment]
Ironically... I'd made these changes in my other branch which is now merged.  Do we keep this around to add policies *also* as @rachaelparris suggested, or call it good?

**Comment by sutt0n** [comment]
I'm adding more @kkoskelin -- marking this as do not merge for now

File: web-api/src/app-public.js [hunk 1]
```diff
@@ -6,6 +6,13 @@ const { lambdaWrapper } = require('./lambdaWrapper');
 const app = express();
 
 app.use(cors());
+app.use((req, res, next) => {
```
Comment (kkoskelin) [inline web-api/src/app-public.js]: I presume this was for testing and you'd like to remove it?

File: web-client/src/views/Interstitial.jsx [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 
 export const Interstitial = connect(
   {
     alertHelper: state.alertHelper,
+    gotoPublicSearchSequence: sequences.gotoPublicSearchSequence,
+    isPublic: state.isPublic,
   },
-  function Interstitial({ alertHelper }) {
+  function Interstitial({ alertHelper, gotoPublicSearchSequence, isPublic }) {
+    console.log('alertHelper', alertHelper);
```
Comment (rachaelparris) [inline web-client/src/views/Interstitial.jsx]: Remove?

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/views/Interstitial.jsx (hunk 2 above)
Comment (sutt0n) [reply web-client/src/views/Interstitial.jsx]: seems my linter missed that -- ty!

**Pull request merged**